hdiamond*.c

```diff
@@ -0,0 +1,40 @@
+/*Lloyd Page*/
```

Comment:
Perfect! 💯 Really appreciate the work you put into this assignment, Lloyd! Keep it up. 😃